module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -723,6 +737,26 @@ public function getILSLoginMethod($target = '')
         return false;
     }
 
+    /**
+     * Connect user authenticated by shibboleth as library card to his account.
```

Comment:
I'd change `user authenticated by shibboleth` to `authenticated user` to make this more generic.